tools/check-swagger

```diff
@@ -1,9 +1,46 @@
 var SwaggerParser = require('swagger-parser');
 
+function checkDuplicateOperationIds(api) {
+    function isDuplicateElement(array, item) {
```

Comment:
We can simplify this to `return array.indexOf(item) >= 0`.  Is there a reason not to use snake_case for our function names?